src/server.js

```diff
@@ -17,15 +18,24 @@ dotenv.config();
 const app = express();
 app.response.sendStatus = formattedSendStatus;
 
+const rateLimit = rateLimiter({
+  windowMs: process.env.RATE_WINDOW * 60 * 1000,
+  max: process.env.RATE_REQ_NUMBER,
+  message: { error: { message: 'Too many requests, try again later!' } },
+  standardHeaders: true,
+  legacyHeaders: false,
+});
+
 app.use(cors({ credentials: true, origin: process.env.WEBAPP_HOST }));
 app.use(express.json());
 app.use(cookieParser());
+app.use(rateLimit);
 
 app.use(oauthRoutes);
 app.use(studentsRoutes);
 app.use(usersRoutes);
 app.use(mealSessionsRoutes);
 
-app.listen(3000, () => {
-  console.log('Server is running on port 3000');
+app.listen(process.env.API_PORT, () => {
```

Comment:
Adicionar fallback pra porta 3000 para simplificar ambiente de desenvolvimento:

```suggestion
app.listen(process.env.API_PORT || 3000, () => {
```